docs/reference/team-processes/triage-process.md

```diff
@@ -146,9 +146,26 @@ If a ticket has already been filed for a previous failure of the same package an
 
 #### Security Warnings
 
-Dependabot will produce [Security Alerts][dependabot-security-alerts] for packages that have registered [CVE][wiki-CVE] numbers that cannot be resolved automatically. At least once during your triage ownership rotation, check these alerts to see if there are any vulnerabilities identified, especially those marked as as critical or high severity. You may also choose review these as a team in the triage meeting if you wish.
+Dependabot will produce [Security Alerts][dependabot-security-alerts] for packages that have registered [CVE][wiki-CVE] numbers that cannot be resolved automatically. At least once during your triage ownership rotation, check these alerts to see if there are any vulnerabilities identified, especially those marked as as critical (or even high) severity. You may also choose review these as a team in the triage meeting if you wish, and it may be good to collaborate with the other triage owner to either make sure you're not working on fixing the same vulnerability, or to pair on a fix.
 
-Be sure to timebox yourself on these fixes, but try to manually upgrade the dependency to the patched version shown in the alert. It may be good to collaborate with the other triage owner to either make sure you're not working on fixing the same vulnerability, or to pair on a fix. Your commit message may begin with something similar to `chore(deps): Security <list affected packages>`.
+If Dependabot thinks it can automatically fix the alert for us, a "Create dependabot security update" button will be present. This is a good clue, as well as a "Dependabot can't update vulnerable dependencies without a lockfile" note, that the upgrade should be simple, but we'll need to manually resolve the alert since Dependabot doesn't have access to our `yarn.lock` file (see Bananafox info above). You can check what packages are affected by running `yarn why <package name>`. If there's only one or two affected packages, consider trying to manually upgrade the dependency to the patched version shown in the alert. 
+
+However, sometimes, security alerts are for deeply nested dependencies. In this case, we use [Yarn resolutions](https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/). To resolve the security warning:
+
+1. Copy the *patched version* from the security warning into the "resolutions" section of our root-level `package.json`, example:
+```
+"resolutions": {
+  "postcss": "^7.0.36",
+```
+
+1. Update the `yarn.lock` file by running `yarn install`
+1. Commit your changes in a PR titled `chore(deps): Security <list affected packages>`
```

Comment:
I generally try to avoid calling out security patches.  Maybe just "Upgrade <packages" would be fine?  It's even worse if the tests don't pass

Reply:
Good point 👍